pydis_site/apps/api/models/bot/infraction.py

```diff
@@ -69,6 +69,14 @@ class Infraction(ModelReprMixin, models.Model):
         help_text="Whether a DM was sent to the user when infraction was applied."
     )
 
+    jump_url_text = models.CharField(
```

Comment:
I was initially doing what you described but changed to this because it looked cleaner bot-side. But I'll do what you're suggesting because it will be more modular.

Reply:
Done.